Dockerfile

```diff
@@ -18,4 +18,11 @@ EXPOSE 5000
 
 FROM base as development
 RUN poetry install
-ENTRYPOINT [ "poetry", "run", "flask", "run", "--host", "0.0.0.0"]
\ No newline at end of file
+ENTRYPOINT [ "poetry", "run", "flask", "run", "--host", "0.0.0.0"]
+
+FROM base as test
+RUN pip3 install pytest
```

Comment:
This line should be irrelevant since you'll get pytest installed anyway when the `poetry install` occurs, and you're using poetry's environment to run pytest